app/assets/stylesheets/3-atoms/_badges.scss

```diff
@@ -2,7 +2,7 @@
   display: inline-block;
   padding: 10px 20px;
   font-weight: 700;
-  line-height: 1;
+  line-height: 0.5;
```

Comment:
I don't think we should be using a line height less than 1.
Once we move the badge out of the H1 tags we would need to relook at the CSS styling for these badges.